gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -954,4 +967,13 @@ private IOStatisticsStoreBuilder createStoreBuilder() {
             });
     return storeBuilder;
   }
+
+  public void clearHttpStats() {
+    instanceIOStatistics.setCounter(GhfsStatistic.ACTION_HTTP_GET_REQUEST.getSymbol(), 0L);
```

Comment:
Let's add an API to clear the statistic, instead of setting to 0 from setCounter